wfapi/wfapi.ipldsch

```diff
@@ -430,4 +430,27 @@ type CatalogMirrorsByPacktype {Packtype:WarehouseAddrList}
 
 type ReplayCapsule union {
 	| Plot "plot.v1"
-} representation keyed
\ No newline at end of file
+} representation keyed
+
+# MirroringConfig defines the mirroring configuration for an entire workspace.
+# This maps the addresses which we want to publish to (of type WarehouseAddr) 
+# to specific configs, which may be backed by various types of storage.
+type MirroringConfig {WarehouseAddr:WarehouseMirroringConfig}
+
+type WarehouseMirroringConfig struct {
+	pushConfig WarehousePushConfig 
+}
```

Comment:
Seems fine to put this here as room for expansion, but I'm curious if you've got any likely examples for future fields that might be added to this struct?  (It's not flunking any vibe checks for me, but I don't have concrete examples in my mind yet either, which seems, ehh.)